.github/workflows/build.yml

```
@@ -0,0 +1,15 @@
+name: build
+
+on:
+  release:
+    types: [ released ]
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish_pure_python.yml@v1
+    with:
+      upload_to_pypi: ${{ (github.event_name == 'release') && (github.event.action == 'released') }}
+    secrets:
+      pypi_token: ${{ secrets.PYPI_PASSWORD_STSCI_MAINTAINER }}
```

@mfixstsci ok! then if you do decide to no longer publish to PyPI, you can also use this workflow just as a build check (if you think that's useful):
```suggestion
    with:
      upload_to_pypi: false
```